docs/source/what_is_jupyter.md

```diff
@@ -0,0 +1,412 @@
+
+A famous computer programmer (Donald Knuth) popularized the idea to combine
+explanatory plain english text with computer code, which is commonly called
+"literate programming". **By adopting this practice, computer programs, as**
+**well as other complex information and ideas, could be better explained to**
+**a wide range of people.**
+
+A program written in this way could be printed on paper and shared by hand
+as an actual "notebook", but in modern times, they are shared digitally as
+"notebook files", and can contain additional rich media like images, 3D models
+and interactive figures, along with data and other program outputs.
+
+By opening a notebook with an editor program like JupyterLab, you can also
+run the code inside the notebook. Since a notebook can contain code that does
+virtually anything, you can do nearly anything regular software can do inside
+of a notebook. For example, a notebook file might be used to:
+
+- Read spreadsheets (or create them) to build reports about your household spending
+- Show proof of your college thesis with interactive graphs and source data
+- Process astronomical data from the James Webb Space Telescope (JWST) (CHECK/LINK)
+- Generate an image of a black hole by processing telescope data (CHECK/LINK)
+- Calculate the presence of gravitational waves from observatory data (CHECK/LINK)
+
+Those last three are real world examples that demonstrate the scientific
+community's usage of computational notebooks (CHECK). Because scientists,
+engineers, and other technical people so often need to perform and communicate
+sophisticated calculations to describe their work, these "computational
+notebooks" became a very popular way to share their work and ideas.
+
+Notebooks are not just for Nasa scientists either: Students, hobbyists, and
+business people commonly use the fast, interactive workflows in JupyterLab to
+make notebooks that help solve everyday problems. You can use them to explain
+your own ideas to others, to learn, to automate tasks at home or at work,
+visualize complex information, and more.
+
+When someone uses the term "notebook", they might be referring to:
+
+- A notebook file on their computer
+- The idea of combining computer code, explanatory text, images and more into
+  the "notebook format"
+- The "Jupyter Notebook" application, used to author and edit digital notebook
+  files
+- Jupyter's ``.ipynb`` notebook file format (used to save your notebook files
+  on your computer), which is interpreted by the ``nbformat`` software library
+
+And the term "Jupyter" might refer to:
+
+- "Project Jupyter", the overarching umbrella project
+- The "Jupyter Notebook" or "JupyterLab" editing programs (or other Jupyter
+  products)
+
+The name Jupyter comes from the three programming languages the project
+originally supported: Julia (ju), Python (pyt) and R (r).
+
+# How do the Jupyter notebook-editor programs work?
+
+It is common for notebook-editor programs like JupyterLab, or Jupyter
```

Comment:
```suggestion
# How do Jupyter notebook editors work?

It is common for notebook editors like JupyterLab, or Jupyter
```

I think "editor program" is overly verbose; most people with text or code writing experience will understand "editor"